packages/nitro-rpc-client/src/serde.ts

```diff
@@ -0,0 +1,260 @@
+import Ajv, { JTDDataType } from "ajv/dist/jtd";
+
+import {
+  ChannelStatus,
+  LedgerChannelInfo,
+  PaymentChannelInfo,
+  RPCRequestAndResponses,
+  RequestMethod,
+} from "./types";
+
+const ajv = new Ajv();
+
+const jsonRpcSchema = {
+  properties: {
```

Comment:
Should we identify any [required properties](https://ajv.js.org/json-schema.html#required)? i.e. `required: ["jsonrpc", "id"]`

Reply:
We are using JSON Type Definitions as opposed to JSON Schema. So all of the properties listed are required: https://ajv.js.org/json-type-definition.html#properties-form